contracts/zero-ex/contracts/src/features/multiplex/MultiplexFeature.sol

```diff
@@ -488,7 +489,7 @@ contract MultiplexFeature is
     // If `i == 0`, the target is the address which should hold the input
     // tokens prior to executing `calls[0]`. Otherwise, it is the address
     // that should receive `tokens[i]` upon executing `calls[i-1]`.
-    function _computeHopTarget(MultiHopSellParams memory params, uint256 i) private view returns (address target) {
+    function _computeHopTarget(MultiHopSellParams memory params, uint256 i) private  returns (address target) {
```

Comment:
Why no longer view?

Reply:
was emitting an event for my test and had forgot to change it back, should be fixed now